src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -43,14 +45,22 @@ class ConsumeMessagesCommand extends Command
     private $logger;
     private $receiverNames;
     private $eventDispatcher;
-
-    public function __construct(RoutableMessageBus $routableBus, ContainerInterface $receiverLocator, EventDispatcherInterface $eventDispatcher, LoggerInterface $logger = null, array $receiverNames = [])
-    {
+    private $servicesResetter;
+
+    public function __construct(
+        RoutableMessageBus $routableBus,
+        ContainerInterface $receiverLocator,
+        EventDispatcherInterface $eventDispatcher,
+        LoggerInterface $logger = null,
+        array $receiverNames = [],
+        $servicesResetter = null
+    ) {
```

Comment:
Please, use only one line, we don't do that in symfony

Reply:
ok :)